packages/react/src/generators/federate-module/lib/utils.ts

```diff
@@ -0,0 +1,157 @@
+import { Tree, getProjects, joinPathFragments } from '@nx/devkit';
+
+import { ensureTypescript } from '@nx/js/src/utils/typescript/ensure-typescript';
+import * as ts from 'typescript';
+
+let tsModule: typeof import('typescript');
+
+if (!tsModule) {
+  tsModule = ensureTypescript();
+}
```

Comment:
i think the import on the top `import * as ts from 'typescript';` already assumes typescript is loaded, then no need to call `ensureTypescript()`. i think we could remove tsModule variable and use ts directly.

Reply:
If it is not loaded wouldn't it fail?
I will switch it to use types from the import `import * as from `typescript` and rename the other functional calls to use `tsModule` instead